openlibrary/templates/check_ins/check_in_form.html

```diff
@@ -26,12 +26,12 @@
   $ day_read = int(split_date[2]) if len(split_date) > 2 else ''
 
 <div class="check-in" data-work-olid="$(work_olid)" data-event-type="3">
-  <form method="dialog" class="check-in__form" action="$(work_key)/check-ins" method="post">
+  <form class="check-in__form" action="$(work_key)/check-ins" method="post">
     <input type="hidden" name="event_id" value="$(event_id)">
     $if edition_key:
       <input type="hidden" name="edition_key" value="$(edition_key)">
     <div>
-      $_('Add an optional check-in date.  Check-in dates are used to track yearly reading goals.')
+      $_('Add an optional check-in date. Check-in dates are used to track yearly reading goals.')
```

Comment:
```suggestion
      $_('Add an optional check-in date.  Check-in dates are used to track yearly reading goals.')
```
I don't really understand why this change was made, and it may affect translations.  Can it be reverted?

Reply:
Probably cannot be reverted, as our workflows made the change to begin with.